unit_tests/glhe/input_processor/test_input_processor.py

```diff
@@ -74,15 +74,15 @@ def test_validate_fluid(self):
 
         self.run_validate(d)
 
-        d = {'fluid': {'fluid-type': 'EA', 'concentration': 100}}
+        d = {'fluid': {'fluid-type': 'EA', 'concentration': 60}}
```

Comment:
scp only has props up to 60% concentration, which in this application is more than plenty. scp was throwing warnings here to this effect, so I backed the concentrations down to eliminate the warnings.